main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAgencySchema.java

```diff
@@ -0,0 +1,35 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+import java.util.Locale;
+import java.util.TimeZone;
+
+
+@GtfsTable("agency.txt")
+public interface GtfsAgencySchema extends GtfsEntity {
+    @FieldType(FieldTypeEnum.ID)
```

Comment:
@aababilov What would you think of adding a `@ConditionallyRequired` annotation for fields like `agency_id` where GTFS has the field as "Conditionally Required"? Looks like this is currently just implemented in `AgencyConsistencyValidator` as a validation rule, but IMHO it would be nice to have some indicator on these fields that they are subject to special treatment (i.e., they aren't "Optional"). Even if the annotation isn't used for generating code I think the indicator would still be helpful for devs to check and make sure that current and future fields are being treated according to the spec.

Reply:
Good point! Done.